files/etc/apache2/sites-available/ssl.conf

```diff
@@ -0,0 +1,132 @@
+<IfModule mod_ssl.c>
+	<VirtualHost _default_:443>
+		ServerAdmin webmaster@localhost
+
+		DocumentRoot /var/www/web
+
+		# Available loglevels: trace8, ..., trace1, debug, info, notice, warn,
+		# error, crit, alert, emerg.
+		# It is also possible to configure the loglevel for particular
+		# modules, e.g.
+		#LogLevel info ssl:warn
+
+		ErrorLog ${APACHE_LOG_DIR}/error.log
```

Comment:
Remember to add a tail of the error-log to etc/service/apache-error-forwarder/run (or introduce an extra service).

(I've avoided tailing the access-log itself for now as developers are mostly interested in errors)

Reply:
The error log goes to the same file as the error log for port 80 so this should be handled already (although I did not know that -- thanks for enlightening me :smile:)